haskell-cabal.el

```diff
@@ -488,7 +489,10 @@ OTHER-WINDOW use `find-file-other-window'."
                           (string= val "{")
                           (not val))
                       (push projectName matches)
-                    (push val matches))))
+                    (push (if (eq 'stack-ghci process-type)
+                              (concat projectName ":" val)
```

Comment:
`projectName` is whatever appears in the 'name:' field at the top of the cabal file, it's in one of the hidden sections in the diff, but `projectName` is defined on line 481 with `(haskell-cabal--get-field "name")`. Stack doesn't have a separate notion of a project to what cabal provides as far as I can tell.